src/main/java/hu/oe/nik/szfmv/automatedcar/visualization/debug/DebugViewer.java

```diff
@@ -0,0 +1,72 @@
+package hu.oe.nik.szfmv.automatedcar.visualization.debug;
+
+import hu.oe.nik.szfmv.automatedcar.visualization.Utils.DrawingInfo;
+
+import java.awt.*;
+
+
+/**
+ * Displays debugging-related information on the screen
+ */
+public class DebugViewer {
+
+    private Graphics2D graphics2D;
+    private DrawingInfo info = new DrawingInfo(Color.GREEN, 2);
+
+    /**
+     * @return The information represening the color and borderline width of the object
+     */
+    public DrawingInfo getInfo() {
+        return info;
+    }
+
+    /**
+     * @param info The information represening the color and borderline width of the object
+     */
+    public void setInfo(DrawingInfo info) {
+        this.info = info;
+    }
+
+    private boolean debuggerSwitchedOn = true;
+
+    /**
+     * @return If debugger is switched on or not
+     */
+    public boolean isdebuggerSwitchedOn() {
```

Comment:
rename to: isDebuggerSwitchedOn

Reply:
good catch, I just autogenerated them and didnt notice

Reply:
resolved: c9ebfbbb773d17e5cbd7e2126d91314275e7bb88